packages/ui-kit/package.json

```diff
@@ -4,10 +4,12 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc -p tsconfig.build.json",
-    "dev": "tsc -p tsconfig.build.json --watch",
+    "clean": "rimraf dist icons",
```

Comment:
This isn't a huge issue, but it might be nice to add a devDependency on rimraf, since I think just transitive right now.